### DISCUSSION TITLE
Subtile and description in blogs

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#1417] [Q&A] [answered]
Hi, 

I want to have a subtitle below my blog's title and a description at the bottom of every page. 

This description is what I want for my whole blog so when one searches it on google could see next my blog's name.  

Not sure I could find the relevant information in the doc. Any help please?

**Comment by dragonstyle** [comment]
- For the subtitle you might consider a `banner` style title block, more here: https://quarto.org/docs/websites/website-listings.html#listing-fields

- For the description at the bottom of every page, you could consider adding a page footer. More here: https://quarto.org/docs/websites/website-listings.html#listing-fields

HTH!

**Reply by ghost** [reply]
Thanks, but in the second link I found no information for the page footer.

**Reply by dragonstyle** [reply]
Sorry - copied wrong link! Try this:

https://quarto.org/docs/websites/website-navigation.html#page-footer

**Reply by ghost** [reply]
So, it worked. My follow-up question is how can I adjust the footer to appear on the left instead of the centre of the page?

**Comment by dragonstyle** [comment, marked as answer]
Try using the left option. 

```website:
  page-footer: 
    left: "Copyright 2021, Norah Jones" 
    right: 
      - icon: github
        href: https://github.com/
      - icon: twitter 
        href: https://twitter.com/  
```

**Reply by ghost** [reply]
Thanks. So the only, thing that has not worked is the subtitle. 

I used a banner and indeed I can create a subtitle, but it looks ugly. I just want a subtitle below the title of my blog. 

![image](https://user-images.githubusercontent.com/26636184/179345235-f49843c8-8fad-4a6f-b62d-d3b398a326a6.png)
